sql/step_68_upgrade_pat_photo_size.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE PATIENT_PROFILE_PHOTO CHANGE COLUMN PAT_PHOTO PAT_PHOTO MEDIUMBLOB NULL DEFAULT NULL;
```

Comment:
There is already a step_68; see the step_04_all_following_steps script which is also a conflict the way it is currently coded.

Reply:
yeah I think there fork was not aligned, @AndreiDodu you should merge the current develop.
anyway why this change here? is not in the OP-438 scope I think...

Reply:
Pardon, I made a mistake here with commits. I will solve it on Saturday/Sunday. I will correct also the other code reviews warnings made by @dbmalkovsky.

Reply:
I returned to the old version. The new commit has only the SMS "module".